schemas/common/trinary-args.json

```diff
@@ -0,0 +1,25 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "$id": "http://jsonlogic.com/schemas/common/trinary-args.json",
+
+  "title": "Trinary Args",
+  "description": "Up to three args of valid JSON Logic data source.",
+
+  "oneOf": [
+    {
+      "title": "Array",
+      "description": "An array with one or three elements.",
+      "type": "array",
+      "minItems": 1,
+      "maxItems": 3,
```

Comment:
Isn't it enough when we provide one-or-more args?

Reply:
I thought so too at the beginning, then I saw that for example `+` and `*` take any number of arguments from  1 to infinity.
But  `-` and `/` take only up to 2 - no more.

And then there are other special ops which take up to 3  like `<` and `<=` (in contrast to `>` and `>=`).

`if` is the most complex number of args, it `2n + 1`, so 1, 3, 5, 7, 9, etc.

To make all of this even more complex, all operators support a single argument without an array.
It even doesn't matter if the provided number of arguments make sense,  `jsonLogic.apply` will try it's best to never fail and output nonsense in the worst case 😅